CHANGELOG.md

```diff
@@ -4,6 +4,20 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+## [1.1.1] - 2023-03-02
+
+### Changed
+- Changed scripts for injecting vars on nginx
+- Changed the user to run in container on values.yml
+- Changed copy command on docker file 
```

Comment:
Hi @fabiodmota,
can you describe why these things changed (the improvements/fixes) instead of the technical place, where something changed please?. Otherwise someone reading the changelog would still need to go through the source to check the implications of upgrading to the new version